packages/suite/src/components/suite/modals/metadata/MetadataProvider/index.tsx

```diff
@@ -1,26 +1,44 @@
 import React, { useState } from 'react';
 import styled from 'styled-components';
-import { Modal, Button } from '@trezor/components';
+import { Modal, P, Button, variables, colors } from '@trezor/components';
+
 import { Translation } from '@suite-components';
 import { useActions } from '@suite-hooks';
 import * as metadataActions from '@suite-actions/metadataActions';
 import { Deferred } from '@suite-utils/deferred';
 import { MetadataProviderType } from '@suite-types/metadata';
-import { resolveStaticPath } from '@suite-utils/nextjs';
+
+const { FONT_SIZE, FONT_WEIGHT, SCREEN_SIZE } = variables;
 
 const Buttons = styled.div`
     display: flex;
+    justify-content: center;
     flex-direction: row;
-    justify-content: space-around;
 `;
 
+// todo: can't use button from @trezor/components directly, probably inconsistent design again
```

Comment:
We should try to re-use components/styles as much as we can to make design updates across the board easier. Not sure how it applies here, so I'll let @slowbackspace chime in :slightly_smiling_face: